packages/i18n/src/locales/en_US.js

```diff
@@ -1251,6 +1251,47 @@ const translations = {
               description: 'Moves the chunk that contains the document specified by the query to the destination shard. uses the moveChunk command',
               example: 'sh.moveChunk(ns, query, destination)',
             },
+            balancerCollectionStatus: {
+              link: 'https://docs.mongodb.com/manual/reference/method/sh.balancerCollectionStatus',
+              description: 'Gets the balancer collection status for a sharded DB. Uses the balancerCollectionStatus command',
+              example: 'sh.balancerCollectionStatus(ns)',
+            },
+            enableBalancing: {
+              link: 'https://docs.mongodb.com/manual/reference/method/sh.enableBalancing',
+              description: 'Enable balancing',
+              example: 'sh.enableBalancing(ns)',
+            },
+            disableBalancing: {
+              link: 'https://docs.mongodb.com/manual/reference/method/sh.disableBalancing',
+              description: 'disable balancing',
+              example: 'sh.disableBalancing(ns)',
+            },
+            getBalancerState: {
+              link: 'https://docs.mongodb.com/manual/reference/method/sh.getBalancerState',
+              description: 'Enables or disables the balancer. Starting in MongoDB 4.2, the method also enables auto-splitting if enabling the balancer and disables auto-splitting if disabling the balancer. ' +
+                'Use sh.getBalancerState() to determine if the balancer is currently enabled or disabled and sh.isBalancerRunning() to check its current state.',
```

Comment:
Is this accurate? I would not expect a method named `getBalancerState()` to enable or disable the balancer

Reply:
Oh good catch! it got mixed up with setBalancerState